node/src/actors/chain_manager/mod.rs

```diff
@@ -2130,9 +2135,11 @@ fn current_committee_size_requirement(
     decreasing_step: u32,
     last_consolidated_checkpoint: u32,
     current_checkpoint: u32,
+    last_checkpoint_signed_by_bootstrap: u32,
 ) -> u32 {
-    // If the last consolidated superblock is 0, return the default committee size
-    if last_consolidated_checkpoint == 0 {
+    assert!(last_consolidated_checkpoint <= current_checkpoint, "Something went wrong as the last consolidated checkpoint is bigger than our current checkpoint {} > {}", last_consolidated_checkpoint, current_checkpoint);
+    // If the last consolidated superblock or the current checkpoint is below last_checkpoint_signed_by_bootstrap, return the default committee size
+    if last_consolidated_checkpoint <= last_checkpoint_signed_by_bootstrap {
```

Comment:
Maybe we could separate in two cases? Something like
- If `last_consolidated_checkpoint = current_checkpoint`, then return the default committee size
- if `last_consolidated_checkpoint < current_checkpoint`, the return the last committee size, since it should be the same as the bootstrap committee size